src/CONST.js

```diff
@@ -124,8 +124,12 @@ const CONST = {
     PRIVACY_URL: 'https://use.expensify.com/privacy',
     LICENSES_URL: 'https://use.expensify.com/licenses',
     PLAY_STORE_URL: 'https://play.google.com/store/apps/details?id=com.expensify.chat&hl=en',
-    ADD_SECONDARY_LOGIN_URL: 'settings?param={%22section%22:%22account%22}',
-    MANAGE_CARDS_URL: 'domain_companycards',
+    ADD_SECONDARY_LOGIN_URL: '/settings?param={%22section%22:%22account%22}',
+    VALIDATE_CODE_URL: (accountID, validateCode, exitTo = '') => {
```

Comment:
This would be better in ROUTES, there are no functions in this file, and tons in ROUTES